script.js

```diff
@@ -0,0 +1,90 @@
+const startButton = document.querySelector('#start');
+const resetButton = document.querySelector('#reset');
+const breakButton = document.querySelector('#break');
+const pomodoroButton = document.querySelector('#pomodoro');
+const timer = document.querySelector('#pomodoro-time');
+let minutes = 25;
+let seconds = 0;
+let isPaused = true;
+let startTimer;
+let mode = "pomodoro";
+
+function format(value) {
+  if (value < 10) {
+    return '0' + value;
+  } else {
+    return value;
+  }
+}
+
+function toggle(minutes, seconds) {
+  clearInterval(startTimer);
+  timer.textContent = `${format(minutes)}:${format(seconds)}`;
```

Comment:
Даша,  тут недостаточно только вывести на страницу
Получается ты отображаешь 25:00 а таймер при нажатии опять на старт , продолжает не с 25 мин , а с того места на кот остановился
Нужно перезаписывать переменные minutes и seconds
```suggestion
function toggle(min, sec) {
  clearInterval(startTimer);
  timer.textContent = `${format(min)}:${format(sec)}`;
  minutes = min;
  seconds = sec;
}
```